src/components/KMultiselect/KMultiselect.vue

```diff
@@ -555,7 +587,7 @@ const numericWidthStyle = computed(() => {
 
 const nonSlimStyle = computed(() => {
   return {
-    width: (numericWidth.value - 30) + 'px',
+    width: (numericWidth.value - 40) + 'px',
```

Comment:
suggestion: can you add a comment here explaining why this is set to a value of `40`?